annotations/views.py

```diff
@@ -251,28 +250,35 @@ def collection_texts(request, collectionid):
 
 
 @ensure_csrf_cookie
-@login_required
 def text(request, textid):
     """
-    Provides the main text annotation view.
+    Provides the main text annotation view for logged-in users.
+    Provides summary of the text for non-logged-in users.
     """
-    template = loader.get_template('annotations/text.html')
+    text_dict = dict()
```

Comment:
Can you give this variable a more informative name? Since you use ``context`` for the ``RequestContext`` on 276, maybe you could call this ``context_data``? Also, the convention for instantiating dicts is ``{}``. For example:

```python
context_data = {}
```

You can populate it with all of the values on 260 - 263 in one pass, e.g.

```python
context_data = {
    'key': value,
    ...
}
```